package.json

```diff
@@ -20,19 +20,19 @@
     "release:publish": "npm publish --registry=https://registry.npmjs.org"
   },
   "devDependencies": {
-    "@babel/core": "^7.22.5",
-    "@babel/eslint-parser": "^7.22.5",
+    "@babel/core": "^7.23.3",
+    "@babel/eslint-parser": "^7.23.3",
     "eslint": "^8.44.0",
```

Comment:
Upgrading "eslint" introduces a bunch of DeprecationWarnings that require dropping support for Eslint@7 to resolve. Will leave that (and the prerequisite conversations) for a separate PR.

Reply:
Note that our CI intentionally runs on both "local" and v7 of eslint: https://github.com/salesforce/eslint-plugin-lwc/blob/bbd1d950652bc040beb2781ef1bcfb4ff28e7777/.circleci/config.yml#L3